src/docfx/build/xref/XrefResolver.cs

```diff
@@ -79,15 +79,35 @@ internal class XrefResolver
         var (xrefError, xrefSpec, resolvedHref) = ResolveXrefSpec(new SourceInfo<string>(uid, href.Source), referencingFile, inclusionRoot);
         if (xrefError != null || xrefSpec is null || string.IsNullOrEmpty(resolvedHref))
         {
-            return (xrefError, null, alt ?? "", null);
+            return (xrefError, null, alt ?? "", null, !string.IsNullOrEmpty(alt));
         }
 
-        var displayPropertyValue = displayProperty is null ? null : xrefSpec.GetXrefPropertyValueAsString(displayProperty);
+        var (displayPropertyValue, localizableOfDisplayProperty) =
```

Comment:
We should probably limit the scope to the `title` property in conceptual pages only. All the other xref syntax is mostly used for references that aren't localizable. Most use of `displayProperty` is to reference non-localizable strings,  e.g. [`displayProperty=nameWithType`](https://github.com/dotnet/docs-archive-pr/blob/8b02460ce2f2775e490ac629959d0a29c51a5f04/docs/framework/windows-identity-foundation/namespace-mapping-between-wif-3-5-and-wif-4-5.md#wif-35-to-wif-45-namespace-map)

Reply:
Yes. We limit the scope here 
![image](https://user-images.githubusercontent.com/67643974/146497846-69cdd449-a1c3-45af-a350-cc781b87fd08.png)